packages/app/src/systems/Error/services/ReportErrorService.tsx

```diff
@@ -0,0 +1,71 @@
+import type { FuelWalletError, ReportErrorFrequency } from '@fuel-wallet/types';
+
+import { errorToFuelError } from '../utils';
+
+import { REPORT_ERROR_FREQUENCY_KEY, REPORT_ERROR_EMAIL } from '~/config';
+import { Storage, db } from '~/systems/Core/utils';
+
+function encodeHTMLEntities(text: string) {
+  const textArea = document.createElement('textarea');
+  textArea.innerText = text;
+  return textArea.innerHTML;
+}
+
+export class ReportErrorService {
+  static async reportErrors() {
+    const errors = await this.getErrors();
+    // send error as an email to the team
+    const errorMailBody = encodeHTMLEntities(
+      errors.map((error) => JSON.stringify(error)).join('\n')
+    ).slice(0, 2000);
+    window?.open(
+      `mailto:${REPORT_ERROR_EMAIL}?subject=Fuel Wallet Error Report&body=${errorMailBody}`,
+      '_blank'
+    );
+    return true;
```

Comment:
why need to return true here? I don't see any place using reportErrors as boolean condition